### ISSUE TITLE
Screen Reader: Group Chat: The button 'Start a chat, refer a friend' is announced as 'referral'

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77457]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe to focus the Create tab and double-tap to activate it
3. Swipe to focus the Start chat button and double-tap to activate it
4. Swipe to focus the "Start a chat, refer a friend" button
5. Note how the element is displayed and announced.

## Expected Result:
The button should be announced with its displayed text, e.g.:
- 'Start a chat, refer a friend,'

## Actual Result:
The button 'Start a chat, refer a friend' is announced as 'referral'.

## Area issue was found in:
Group chat

## Failed WCAG checkpoints
WCAG 2.5.3 Label in Name
https://www.w3.org/WAI/WCAG22/Understanding/label-in-name

## User impact:
Speech-input users can directly activate controls on a page with fewer surprising changes of focus. Text-to-speech users will have a better experience because the labels that they hear match the visible text labels that they see on the screen.

## Suggested resolution:
Update the accessibilityLabel to match the visible text and ensure it is announced with the visible text:
- 'Start a chat, refer a friend'
https://appt.org/en/docs/react-native/samples/accessibility-label
https://reactnative.dev/docs/accessibility#accessibilitylabel

## Environment:
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

![Image](https://github.com/user-attachments/assets/7d5485de-f42e-446e-a6f4-7209b3ef3b0a)

https://github.com/user-attachments/assets/32baa744-d960-4c79-a19d-74f5a1914a57

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7007944
**Issue reported by:** NA
**Slack conversation:** NA
